eng/Version.Details.xml

```diff
@@ -73,9 +73,9 @@
       <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-runtime</Uri>
       <Sha>5535e31a712343a63f5d7d796cd874e563e5ac14</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NET.Runtime.WorkloadTesting.Internal" Version="9.0.0-preview.3.24151.4">
+    <Dependency Name="Microsoft.NET.Runtime.WorkloadTesting.Internal" Version="9.0.0-preview.3.24158.8">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>50d6e5d5ffd05dd4034cffd222ea610baedcc326</Sha>
+      <Sha>54d318d6da76e409d4e865220d9923283d55a06d</Sha>
```

Comment:
Do we have this set up on a maestro auto-update?

Reply:
After this PR, I can set that up. I wanted to get the latest changes in first.